code/modules/mob/living/carbon/human/species.dm

```diff
@@ -285,6 +285,10 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 /datum/species/proc/check_roundstart_eligible()
 	if(id in (CONFIG_GET(keyed_list/roundstart_races)))
 		return TRUE
+	// SPLURT EDIT BEGIN
+	if(roundstart == TRUE)
```

Comment:
I think the config should be authoritative when it comes to roundstart, honestly, and the fewer core-code edits this PR adds, the better.